src/components/MultiSelect/MultiSelect.mdx

```diff
@@ -27,6 +27,7 @@ import { changelogData } from "./multiSelectChangelogData";
 - {<Link href="#block-element-or-float" target="_self">Block Element or Float</Link>}
 - {<Link href="#width" target="_self">Width</Link>}
 - {<Link href="#default-open-state" target="_self">Default Open State</Link>}
+- {<Link href="#close-on-blur-state" target="_self">Close on Blur State</Link>}
```

Comment:
Nit: the link doesn't work. It should be `#closeOnBlur` to match the section's title.

Reply:
On line 581, rename `## closeOnBlur` to `## Close on Blur State` to match the section name in `MultiSelectGroup.mdx`.